libs/src/main/python/dlpx/virtualization/libs/_logging.py

```diff
@@ -17,4 +18,5 @@ class PlatformHandler(Handler):
     """
     def emit(self, record):
         msg = self.format(record)
+        msg = to_str(msg)
```

Comment:
Should we be converting to a character string BEFORE calling into the format function?

Reply:
`record` is not a string or byte string which I checked because I had the same thought. But now that you mention it, yeah this should be moved up. We can just convert the message directly on the record object before formatting it and that should be sufficient

Reply:
Forgot to update this thread but ended up leaving this as we had it originally with the `to_str` after the format call. Although really we technically shouldn't need to as that call uses record.getMessage(), which always converts to a str before returning anyway. Leaving our conversion though in case anything were to change in the future with that behavior